src/main/java/uk/ac/ebi/ait/filecontentvalidatorservice/FileContentValidatorServiceApplication.java

```diff
@@ -1,12 +1,39 @@
 package uk.ac.ebi.ait.filecontentvalidatorservice;
 
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+import uk.ac.ebi.ait.filecontentvalidatorservice.service.FileContentValidationHandler;
 
 @SpringBootApplication
-public class FileContentValidatorServiceApplication {
+@RequiredArgsConstructor
+@Slf4j
+public class FileContentValidatorServiceApplication implements ApplicationRunner {
+
+	@NonNull
+	private FileContentValidationHandler fileContentValidationHandler;
+
+	private static final String FILE_PATH_OPTION = "fileContentValidator.files";
 
 	public static void main(String[] args) {
```

Comment:
Any reason for not creating a PID file like the rest of the services?

Reply:
I am not sure this application would need one. This is a command line application and not monitored by monit. It launched by the file processing listener app. After doing his validation and message sending it exits.
@haseeb-gh Do you see any reason it would need its own PID file?